build-tools/create-packs/Directory.Build.targets

```diff
@@ -98,6 +106,7 @@
       <_WLTemplates Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nupkgs\Microsoft.Android.Templates.*.nupkg" />
       <!-- Runtime packs are not yet supported by workloads -->
       <!-- <_WLPacks Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nupkgs\Microsoft.Android.Runtime.*.nupkg" /> -->
+      <_AotPacks Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nupkgs\microsoft.netcore.app.runtime.aot.*.cross.*.nupkg" />
```

Comment:
Why is this neede?

Reply:
For our local build, this downloads the pack and places it in `~/android-toolchain/dotnet/packs`.